### PR TITLE
removes obsolete CLI arguments

### PR DESCRIPTION
**Pull request by joemfb**
also sorts them and performs some general cleanup in anticipation of the cc-release merge.

### CODE REVIEW

**Comment by ohAitch** [comment]
Is -X not actively used for ship surgery?

**Comment by joemfb** [comment]
I believe not actively, but it has been in the past. As I said, these changes are anticipating a cc-release merge, and any such functionality would need to be re-implemented (and probably rethought) there. I'm comfortable removing this entirely to that end, but I could see an argument that these changes should be limited to a release-candidate-prep-merge branch ...

**Comment by joemfb** [comment]
@benjamin-tlon, bumping for review

**Comment by benjamin-tlon** [comment]
*reviews*

**Comment by benjamin-tlon** [comment]
Why keep `raf_c` etc around if the flags are going away? Is this a temporary change?

**Comment by joemfb** [comment]
I don't want to mess with the internals of `raft.c`, wherein there be dragons.

**Comment by joemfb** [comment]
I should add that we're not using the actual raft implementation (which I think is incomplete?), and so there's no requirement that the arguments be there. But the raft internals dispatch off of those members of the `u3_opts` struct. `raft.c` goes away on cc-release.

File: vere/main.c [hunk 1]
```diff
@@ -189,20 +189,13 @@ _main_getopt(c3_i argc, c3_c** argv)
...
-    return c3n;
-  }
-
-  if ( 0 != u3_Host.ops_u.fak_c ) {
     u3_Host.ops_u.has = c3y;  /* no battery hashing on fake ships. */
     u3_Host.ops_u.net = c3n;  /* no networking on fake ships. */
+    u3_Host.ops_u.nuu = c3y;
   }
 
+  c3_t imp_t = ( (0 != u3_Host.ops_u.who_c) && (4 == strlen(u3_Host.ops_u.who_c)) );
```
Comment (benjamin-tlon) [inline vere/main.c]: `imp_t`?

Comment (benjamin-tlon) [reply vere/main.c]: Maybe a comment?

Comment (benjamin-tlon) [reply vere/main.c]: "Important"? A galaxy?

Comment (joemfb) [reply vere/main.c]: Imperial. See also `imp_c`/`imp_y` in various places.

Comment (ohAitch) [reply vere/main.c]: imperator :)

Comment (joemfb) [reply vere/main.c]: I was close ;)

**Pull request merged**